db/connect.js

```diff
@@ -1,12 +1,7 @@
-const mongoose = require('mongoose')
+const mongoose = require('mongoose');
 
 const connectDB = (url) => {
-  return mongoose.connect(url, {
-    useNewUrlParser: true,
-    useCreateIndex: true,
-    useFindAndModify: false,
-    useUnifiedTopology: true,
-  })
-}
+  return mongoose.connect(url, {});
+};
```

Comment:
why was the options deleted? just making sure this was as intended

Reply:
They were no longer supported with the latest version of Mongoose (https://stackoverflow.com/questions/68958221/mongoparseerror-options-usecreateindex-usefindandmodify-are-not-supported)